packages/backend/src/index.ts

```diff
@@ -72,6 +72,10 @@ import { DefaultEventBroker } from '@backstage/plugin-events-backend';
 import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
 import { MeterProvider } from '@opentelemetry/sdk-metrics';
 import { metrics } from '@opentelemetry/api';
+import dns from 'node:dns';
+
+// Fix https://github.com/backstage/backstage/issues/19931
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
It [looks](https://nodejs.org/api/dns.html#dnssetdefaultresultorderorder) like this setting is global (or at least per-thread) -- if folks like the approach (I leave it to maintainers with broader perspective than mine), then it might make sense to put this setting in the [`create-app` template](https://github.com/backstage/backstage/blob/master/packages/create-app/templates/default-app/packages/backend/src/index.ts) as well.

Reply:
Does the template not use this package? I would assume just using the backend package would trigger this line...

Reply:
They are separate -- for example, the sample app has many plugins installed for testing, but an freshly-scaffolded app is more "blank". Often changes like this need to be duplicated in the templates, but as I said, this whole workflow is about local development so a fix like this might not need to be part of the end-user templates.

Reply:
Indeed `packages/backend` is just a test backend that we use ourselves for local testing, and has nothing at all to do with backends that end users see.

Reply:
How likely is it for production deployments to use `kubectl proxy`? is it ok to assume this is only used in testing environments & product demos?

Reply:
No those aren't released as packages, so they typically don't get any changesets. The only people potentially affected are those that do local development _on the oss repo_ specifically, and only if they run the example backend in the first place instead of doing local testing of their specific plugin.